weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -1050,8 +1050,8 @@ MyApplet.prototype = {
       case 37:/* isolated thunderstorms */
         return _('Isolated thunderstorms')
       case 38:/* scattered thunderstorms */
-      case 39:/* scattered thunderstorms */
         return _('Scattered thunderstorms')
+      case 39:/* scattered thunderstorms */
```

Comment:
The comment ("thunderstorms") is misleading, since case 39 falls through to "showers" with this change.

Maybe change it to match [L937](https://github.com/linuxmint/cinnamon-spices-applets/pull/749/files#diff-c22f045b2183af6df92fd08bf4f76ad0L937)?

```js
case 39:/* http://developer.yahoo.com/forum/YDN-Documentation/Yahoo-Weather-API-Wrong-Condition-Code/1290534174000-1122fc3d-da6d-34a2-9fb9-d0863e6c5bc6 */
```